src/dotnet/App.Maui/MauiDiagnostics.cs

```diff
@@ -41,9 +41,8 @@ public static IServiceCollection AddMauiDiagnostics(this IServiceCollection serv
     {
         services.AddSingleton<Disposer>();
         services.AddTracer(Tracer); // We don't want to have scoped tracers in MAUI app
-        services.AddSingleton(LoggerFactory);
-        services.Add(ServiceDescriptor.Singleton(typeof(ILogger<>), typeof(Logger<>)));
-        /*
+        // services.AddSingleton(LoggerFactory);
```

Comment:
Why is that commented out?

Reply:
@alexyakunin i have commented out this code because it doesn't work, and i don't understand why

Reply:
@crui3er maybe you can check what's going on there? Or it's just on iOS? I would test it on Windows app first to find out if it's an iOS-specific issue.

Reply:
> @alexyakunin i have commented out this code because it doesn't work, and i don't understand why

What does it mean 'doesn't work'? On Android and Window logging does work.